test/e2e/marketplace_test.go

```diff
@@ -0,0 +1,186 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	apis "github.com/operator-framework/operator-marketplace/pkg/apis"
+	operator "github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
```

Comment:
Is the naming for clarity when reading the code as I don't see another `test` package being imported here?